### DISCUSSION TITLE
CustomTkinterTitlebar

### DISCUSSION THREAD

**Discussion by littlewhitecloud** [Akascape/tkinter-toolkit#3] [Add this]
### Screenshot: ![image](https://github.com/littlewhitecloud/CustomTkinterTitlebar/assets/71159641/30f7c785-d189-4865-8095-384259e9b419)

**Comment by Akascape** [comment]
@littlewhitecloud 
I have explored your library before, it looks great tbh.
I will surely add it. I have also made [py-window-styles](https://github.com/Akascape/py-window-styles) and a tweaked [ctktitlebar](https://github.com/Akascape/CTkMenuBar#ctktitlemenu) (similar to your project).
Edit:
I tested your library again, now it works properly in windows 11 (the corners). But I noticed some bugs:
(I tested the tab example)
- The close button is not working on IDLE mode.
- There is a black strip on the top. I guess it can be removed using dwma_attributes.

**Reply by littlewhitecloud** [reply]
Thanks for you checking it out!
If you can, can you make a pr to fix the black strip(Actually it is 6px white frame)?
Thank you very much! Really love this project!
(Also, I noticed that tkinter-toolkit window gets no-responding easily, it is a bug?)
![image](https://github.com/Akascape/tkinter-toolkit/assets/71159641/ae4be7f9-34ec-4eb6-a5a2-dc97686c746e)

**Comment by m7mdna3nou3** [comment]
@littlewhitecloud 
Your Library is great but there is a bug where if I set the attribute of the screen to ```('-fullscreen', True)``` then set it to ```('-fullscreen', False)```
it returns the main title bar and under it the custom title bar, and when I set the ```win.overrideredirect(True)``` I cannot use the title bar to move the window around as I want so is there a fix for that?
![Screenshot 2024-08-04 134140](https://github.com/user-attachments/assets/ab6ec02d-e70f-4f26-99fb-b7dd700ca7c4)
![Screenshot 2024-08-04 134213](https://github.com/user-attachments/assets/173e6820-c7f8-40da-ba49-1fa0805aa8ae)

**Reply by Akascape** [reply]
@m7mdna3nou3 You can also try these libraries if you are doing some titlebar operations: [pywinstyles](https://github.com/Akascape/py-window-styles) and [hPyT](https://github.com/Zingzy/hPyT)

**Reply by m7mdna3nou3** [reply]
I will check them out thanks!!
